src/views/teachers/landing/landing.scss

```diff
@@ -105,87 +83,29 @@ $story-width: $cols3;
         align-items: flex-start;
 
         div {
-            max-width: $cols6;
+            max-width: 24rem;
```

Comment:
Are we deliberately moving away from using these `$cols` variables? Just checking!

Reply:
Yep! Carl and I switched away from it yesterday when we were trying to make it better match the spec.